backend/FwLite/LcmCrdt.Tests/MiniLcmTests/BasicApiTests.cs

```diff
@@ -1,9 +1,47 @@
-﻿namespace LcmCrdt.Tests.MiniLcmTests;
+﻿using Xunit.Abstractions;
 
-public class BasicApiTests(MiniLcmApiFixture fixture): BasicApiTestsBase, IClassFixture<MiniLcmApiFixture>
+namespace LcmCrdt.Tests.MiniLcmTests;
+
+public class BasicApiTests(ITestOutputHelper output): BasicApiTestsBase
 {
+    private readonly MiniLcmApiFixture _fixture = new();
+    public override async Task InitializeAsync()
+    {
+        _fixture.LogTo(output);
+        await _fixture.InitializeAsync();
```

Comment:
yeah I suppose I could make an overload, InitializeAsync comes from a xunit interface, so I can't simply change it because it's being called directly by xunit in other places

Reply:
yeah I don't think I'll bother, since I'd want to keep around the LogTo method anyway